### ISSUE TITLE
Upgrade PyChromecast ?

### ISSUE THREAD

**Issue by chapmanjacobd** [alexdelorenzo/cast_control#26]
```
Traceback (most recent call last):
  File "/home/xk/.local/bin/cast_control", line 5, in <module>
    from cast_control.cli import cli
  File "/home/xk/.local/lib/python3.11/site-packages/cast_control/cli.py", line 10, in <module>
    from .base import RC_NO_CHROMECAST, LOG_LEVEL, NAME, \
  File "/home/xk/.local/lib/python3.11/site-packages/cast_control/base.py", line 12, in <module>
    from pychromecast.controllers.media import MediaStatus
  File "/home/xk/.local/lib/python3.11/site-packages/pychromecast/__init__.py", line 14, in <module>
    from . import socket_client
  File "/home/xk/.local/lib/python3.11/site-packages/pychromecast/socket_client.py", line 23, in <module>
    from . import cast_channel_pb2
  File "/home/xk/.local/lib/python3.11/site-packages/pychromecast/cast_channel_pb2.py", line 33, in <module>
    _descriptor.EnumValueDescriptor(
  File "/home/xk/.local/lib/python3.11/site-packages/google/protobuf/descriptor.py", line 796, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```

**Comment by alexdelorenzo** [comment]
Thanks for opening a issue, I've got PyChromecast 14.0.x support on a dev branch that I'll merge with main soon :)

**Comment by alexdelorenzo** [comment]
Merged support for `PyChromecast>=14.0.1`

**Comment by yochananmarqos** [comment]
@alexdelorenzo Where? There's neither a new tag nor a new release. Also I see there is now pricing. I guess I'll have to drop the AUR package.

**Comment by alexdelorenzo** [comment]
@yochananmarqos Added new PyChromecast support to `master`, I haven't made a release yet. There are changes I upstreamed to the PyChromecast project that were merged but haven't been released yet. Until there's a new release of PyChromecast, YouTube casting support will be broken so I hesitate to make a full release.

This project is free and will always remain free and open source. Nothing has changed for it, nor will anything change in the future.

I appreciate your work packaging this for the AUR, and I know others do, too. No matter your decision, thank you for your time and work. It was a cool thing to do.

Cast Control Desktop is a separate project with a GUI that targets additional hardware, like Fire TV and Roku, and platforms, like Jellyfin and Plex, with support for macOS and Windows. Work on the GUI has directly benefited this project, and will always continue to.

I wish I could give everything away for free, but the truth is my open source contributions have been to my detriment. Health-wise, and otherwise, I cannot continue giving so much of my time away for free. My hope is that separate projects like CCD can help give me the time and resources to continue open source work, and so far that has been the case. CCD has given me the time to work on this project and others.
